### ISSUE TITLE
Question about train_loader.

### ISSUE THREAD

**Issue by lbh666** [vye16/shape-of-motion#60]
Hi, thank you for your well-organized code. I'm a bit confused about why the data loader doesn't shuffle during training. Is this intentional or a possible bug? It's possible that shuffling could introduce more data diversity and further enhance the performance of the method.

**Comment by qianqianwang68** [comment]
Hi I believe the shuffling is achieved manually [here](https://github.com/vye16/shape-of-motion/blob/main/flow3d/data/iphone_dataset.py#L607). Closing the issue now and feel free to reopen it.
